panel/src/components/Views/Files/VideoFilePreview.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+	<div class="k-video-file-preview">
+		<k-file-preview-thumb :options="options">
```

Comment:
`Suggestion`: Looking at this in the new context, I was wondering if we should maybe use `k-file-preview-frame` instead. It could maybe be a little more universal instead of thumb.

Reply:
I agree, will commit the change directly to `v5/develop`